account-number-protector/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Account Number Protector</title>
+</head>
+<body>
+
+  <script>
+
+    //create an empty storage to hold the new results
+    //loop through the given input of 16 numbers
+    //find the index that will start at the last 4 digits and assign to a variable
+    //add 2 asterisks with the new variable and assign to the empty storage
+    //return the new values from empty storage
+
+    function accountNumberProtector(cardNumberString){
+      var results = [];
+      for (var i = 0; i < cardNumberString.length; i++) {
+        var digits = cardNumberString.slice(12);
```

Comment:
Based on the fact that you are using the slice method here, I don't think you need to have a loop in the solution. The `slice` method will utilize a loop to do what it needs to, but for you as the developer, you just need to call the method once, and it will do it's job :+1: